daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/SpecifiedLength2.scala

```diff
@@ -460,6 +470,102 @@ class ElementUnusedUnparser(
 
 }
 
+/**
+ * This is basically a simplified version of the ElementUnusedUnparserSuspendableOperation.
+ * For explicit length choices, we do not need to handle length expressions, choiceLength is
+ * simply a non-negative integer.
+ */
+class ChoiceUnusedUnparserSuspendableOperation(
+  override val rd: ModelGroupRuntimeData,
+  targetLengthInBits: Long)
+  extends SuspendableOperation
+  with StreamSplitter
+  with SkipTheBits {
+
+  private var zlStatus_ : ZeroLengthStatus = ZeroLengthStatus.Unknown
+
+  private var maybeDOSStart : Maybe[DataOutputStream] = Maybe.Nope
+  private var maybeDOSEnd : Maybe[DataOutputStream] = Maybe.Nope
+
+
+  def captureDOSStartForChoiceUnused(state: UState): Unit = {
+    val splitter = new RegionSplitUnparser(rd)
```

Comment:
Missing file for this? I don't see RegionSplitUnparser anywhere.

Reply:
Added the missing file.